stdlib/2/os/path.pyi

```diff
@@ -4,7 +4,8 @@
 # based on http://docs.python.org/3.2/library/os.path.html
 # adapted for 2.7 by Michal Pokorny
 
-from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr
+from typing import overload, List, Any, Tuple, BinaryIO, TextIO, TypeVar, Callable, AnyStr, Union
+EitherStr = Union[bytes, unicode]
```

Comment:
Actually there's a semantic difference called out in PEP 484, in stubs only, between an outright definition (which is exported) and something that's imported (which is not exported, unless it uses the "X as X" form of import). So as this stub stands it defines a symbol EitherStr that is accidentally usable in type comments like this:
```python
import os.path
foo = ""  # type: os.path.EitherStr
```
This should not be allowed.